test/strategies/node.ts

```diff
@@ -132,6 +168,14 @@ describe('Node', () => {
       assertHasUpdate(updates, 'package-lock.json', PackageLockJson);
       assertHasUpdate(updates, 'npm-shrinkwrap.json', PackageLockJson);
       assertHasUpdate(updates, 'samples/package.json', SamplesPackageJson);
+      // Ensure that the packageName is used vs., the component
+      // when updating samples/package.json:
+      for (const update of updates) {
+        if (update.path === 'samples/package.json') {
```

Comment:
The `assertHasUpdate()` method (line 170) returns the `Update` if found. This would eliminate the need to fin